gruntfile.js

```diff
@@ -3,43 +3,16 @@
 /**
  * Module dependencies.
  */
-var _ = require('lodash'),
-  defaultAssets = require('./config/assets/default'),
-  testAssets = require('./config/assets/test'),
-  fs = require('fs'),
-  path = require('path'),
-  childProcess = require('child_process');
+var _ = require('lodash');
+var defaultAssets = require('./config/assets/default');
+var testAssets = require('./config/assets/test');
+var fs = require('fs');
+var path = require('path');
+var childProcess = require('child_process');
 
 module.exports = function(grunt) {
-  // Project Configuration
-  var LOGO = '';
-  if (process.env.ENVIRONMENT === 'MEM') {
-    LOGO = 'modules/core/client/img/brand/mem-logo-inverted.png'; // EAO Logo
-  } else {
-    LOGO = 'modules/core/client/img/brand/eao-banner-img-lg.png'; // BC Logo
-  }
-  var ENV = '';
-  if (process.env.ENVIRONMENT) {
-    ENV = process.env.ENVIRONMENT;
```

Comment:
The LOGO png doesnt exist.
The ENV is used, but it is always set to "EAO" so I added it to the constants.js file directly.

Reply:
The ENV change came up because the tests need it to be set, but it was only being set in the dist folder when you go to package everything.